lib/workspace.rb

```
@@ -0,0 +1,70 @@
+require 'httparty'
+require 'awesome_print'
+require 'pry'
+require 'dotenv'
+Dotenv.load
+
+require_relative 'slack_api_error'
+require_relative 'recipient'
+require_relative 'channel'
+require_relative 'user'
+
+class Workspace
+  attr_reader :users, :channels, :selected
+  
+  def initialize
+    @users = User.list
+    @channels = Channel.list 
+    @selected = nil
+  end
+  
+  def select_channel(input: nil)
+    channel_names = @channels.map { |channel| channel.name }
+    channel_ids = @channels.map { |channel| channel.slack_id }
+    
+    if channel_names.include?(input)
+      @selected = @channels.find { |c| input == c.name }
+    elsif channel_ids.include?(input)
+      @selected = @channels.find { |c| input == c.slack_id }
+    else
+      return nil
```

I'd like to see this `raise` instead of just returning `nil`.